app/src/main/java/co/condorlabs/customcomponents/helper/masks/PhoneNumberTextWatcherMask.kt

```diff
@@ -17,54 +17,95 @@
 package co.condorlabs.customcomponents.helper.masks
 
 import android.text.Editable
-import android.widget.EditText
-import co.condorlabs.customcomponents.*
-import co.condorlabs.customcomponents.helper.TextWatcherAdapter
+import android.text.TextWatcher
+import co.condorlabs.customcomponents.CHAR_TO_DESCRIBE_A_NUMBER
+import co.condorlabs.customcomponents.EMPTY
+import co.condorlabs.customcomponents.ZERO
 
-class PhoneNumberTextWatcherMask(private val receiver: EditText) : TextWatcherAdapter() {
+/**
+ * @author Alexis Duque on 2020-01-22.
+ * @company Condor Labs.
+ * @email eduque@condorlabs.io.
+ */
+class PhoneNumberTextWatcherMask(
+    private val mask: String,
+    val cursorPos: (Int) -> Unit
```

Comment:
This name is not `intention-revealing name` (Clean Code Chapter 2: Meaningful Names).

Reply:
Renamed to _cursorPosition_

Reply:
Sorry I comment again but it's not enogh for me.
The `cursorPosition` name seems to me like a numeric variable with the position of the cursor. 
However, it's not a variable, it is a event the WatcherMask is calling to notify the change of the cursor position.
In addition, the name should be a verb or phrase verb like `onCursorPositionChanged` or something according to Android standar.

Reply:
Renamed to `onCursorPositionChangeRequired`